### ISSUE TITLE
gd is gd2 on Windows

### ISSUE THREAD

**Issue by leofeyer** [shivammathur/setup-php#80]
This is not a big issue, but you might want to standardize this.

**To Reproduce**

```
runs-on: windows-latest
steps:
    - name: Setup PHP
      uses: shivammathur/setup-php@master
      with:
          php-version: 7.3
          extension-csv: dom, filter, gd, hash, intl, json, pcre, pdo, zlib
          coverage: none
```

**Log**

```
Run shivammathur/setup-php@master
pwsh C:\hostedtoolcache\windows\win32.ps1 -version 7.3 -dir d:\a\_actions\shivammathur\setup-php\master\lib

==> Setup PhpManager 
√ PhpManager Installed 

==> Setup PHP and Composer 
√ PHP Switched to PHP7.3 
√ Composer Installed 

==> Setup Extensions 
√ dom Enabled 
√ filter Enabled 
✗ gd Could not find gd for PHP7.3 on PECL 
√ hash Enabled 
√ intl Enabled 
√ json Enabled 
√ pcre Enabled 
√ pdo Enabled 
√ zlib Enabled 

==> Setup Coverage 
√ none Disabled Xdebug and PCOV 
```

**Workaround**

This is due to the fact that the .dll file is called `php_gd2.dll` on Windows, so I tried this:

```
runs-on: windows-latest
steps:
    - name: Setup PHP
      uses: shivammathur/setup-php@master
      with:
          php-version: 7.3
          extension-csv: dom, filter, gd2, hash, intl, json, pcre, pdo, zlib
          coverage: none
```
**Log**

```
==> Setup Extensions 
√ dom Enabled 
√ filter Enabled 
√ gd2 Enabled 
✗ gd2 Could not find gd2 for PHP7.3 on PECL 
√ hash Enabled 
√ intl Enabled 
√ json Enabled 
√ pcre Enabled 
√ pdo Enabled 
√ zlib Enabled 
```

As you see, the `gd2` extension has now been enabled, however, the script also tried to download it from PECL. Maybe you can unify this, so the script automatically maps `gd` to `gd2` on Windows?

**Comment by shivammathur** [comment]
@leofeyer Fixed in #81. `extension-csv: gd` should work now on all OS.

**Comment by leofeyer** [comment]
Man, you are fast! 🚀 I can confirm that it works now. Thank you very much.
